kernel/kernel-config-4.9.x

```diff
@@ -1,6 +1,6 @@
 #
 # Automatically generated file; DO NOT EDIT.
-# Linux/x86 4.9.54 Kernel Configuration
+# Linux/x86 4.9.60 Kernel Configuration
```

Comment:
I agree with @jodh-intel - if you did this upgrade against a new kernel (and took a defconfig or allyes or similar), and/or you need to have the latest kernel for a feature, can you do the version upgrade as a separate commit (and possibly a separate PR?).

Reply:
We are currently using kernel 4.9.60, but our config file still contains 4.9.54!
I can put this change in a separate PR itself.

Reply:
This sounds like a bit of a "process gap".

@grahamwhaley, @jcvenegas, @erick0z  - the packaging scripts run `make oldconfig`, but shouldn't we be committing the changes to avoid this sort of issue? Ideally, we'd build both `.deb` and `.rpm` packages, diff the two `make oldconfig`-updated `.config` files (they should be the same we'd hope!) then commit one of them as https://github.com/clearcontainers/packaging/blob/master/kernel/kernel-config-4.9.x so the comment line above is always in sync with the kernel version we're building?